docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -1,6 +1,6 @@
 = Installation on cloudscale.ch
-:ocp-minor-version: 4.7
-:ocp-patch-version: {ocp-minor-version}.0
+:ocp-minor-version: 4.8
+:ocp-patch-version: {ocp-minor-version}.2
```

Comment:
```suggestion
:ocp-patch-version: {ocp-minor-version}.9
```

Reply:
As https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.9/rhcos-4.8.9-x86_64-openstack.x86_64.qcow2.gz doesn't exit, I wouldn't do that. 

I'm not quite sure how Red Hat decides which versions are available there, but the most recent is `4.8.2`. Did the download link change?